local/assets/cloud-config/etcd-proxy.yml

```diff
@@ -0,0 +1,84 @@
+#cloud-config
```

Comment:
not currently used, but should be used by the 4th machine in the core-cloud demo `:24`